app/redhat_insights/appserver/static/javascript/views/app.js

```diff
@@ -1,7 +1,25 @@
 import { validate_uuidv4 } from "./util.js";
 
-define(["react", "splunkjs/splunk", "splunkjs/mvc", "app/views/setup"], function (react, splunk_js_sdk, mvc, Setup) {
+export const appNamespace = {
+  owner: "Red Hat",
+  app: "redhat_insights",
+  sharing: "app",
+};
+
+define(["react", "splunkjs/splunk", "splunkjs/mvc", "app/views/setup", 'app/views/setup_configuration'], function (react, splunk_js_sdk, mvc, Setup, Config) {
+  console.debug("Entering app.js");
+
   const e = react.createElement;
+  // Initialize a Splunk Javascript SDK Service instance
+  const http = new splunkjs.SplunkWebHttp();
+  const service = new splunkjs.Service(
+      http,
+      appNamespace,
+  );
+
+  // Validate if the app is already configured
+  Config.is_app_configured(splunk_js_sdk);
```

Comment:
Please move this and those Splunk services to appropriate React component so it isn't called on import.